contrib/container/Dockerfile

```diff
@@ -149,7 +149,7 @@ EXPOSE 5173
 # Install packages required for building python packages
 RUN ./install_build_packages.sh
 
-RUN pip install --require-hashes -r base_requirements.txt --no-cache
+RUN pip install --user --require-hashes -r base_requirements.txt --no-cache
```

Comment:
Why is the install context changed? This means running under a different user id will break the container